main.cpp

```diff
@@ -96,10 +106,26 @@ struct Numeric
     using Type = Temporary<NumType>;
  private: 
     std::unique_ptr<Type> value;
+ 
  public:
 
-    Numeric(Type nuNum) : value( std::make_unique<Type>(nuNum) ) { } 
+    Numeric(NumType nuNum) : value( std::make_unique<Type>(nuNum) ) { }
+    ~Numeric(){ }
 
+    //rule of 5 funcs
+    Numeric(Type&& num){ *value = std::move(num); }
+    Numeric& operator=(NumType&& num)
+    { 
+        *value = std::move(num);
+        return *this;
+    }
```

Comment:
These constructors are being passed the wrong type to conform with the `Rule of 5`. 
Think about how the copy constructor and copy-assign functions work, and what Type is passed to them. 
- [ ] revise